toolchain/install/install_paths.h

```diff
@@ -83,8 +83,9 @@ class InstallPaths {
   // fails for any reason, it will `CARBON_CHECK` fail with the error message.
   static auto MakeForBazelRunfiles(llvm::StringRef exe_path) -> InstallPaths;
 
-  // Provide an explicit install paths prefix. This is useful for testing or for
-  // using Carbon in an environment with an unusual path to the installed files.
+  // Provide an explicit install paths prefix, which must be absolute. This is
+  // useful for testing or for using Carbon in an environment with an unusual
+  // path to the installed files.
```

Comment:
Done. (note, the above paragraph is just a re-wrap)

Reply:
Actually, moved the comment to `prefix()` if you're okay with that. Just trying to avoid adding to the already-long class comment.

I imagine the re-wrap spot will still be apparent.

Reply:
On one hand, I like the comment on prefix.

I still somewhat feel like it might be relevant to have a class level comment because this impacts when you should call some of the factory functions...

But not really blocking here.

Reply:
Moved to the class. I'll send you a separate change with suggested cleanups for the class comment.